### PR TITLE
PERF-5042: added device_group

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Comment by LoryApiFortress** [comment]
@paweldudzinski Can you please review? It covers [PERF-5042](https://saucedev.atlassian.net/browse/PERF-5042) Thanks

File: docs/dev/api/insights.md [hunk 1]
```diff
@@ -2874,6 +2881,12 @@ Return an array of test cases (grouped by name) with statistical details.
      <td><p><small>| QUERY | OPTIONAL | ARRAY OF STRINGS |</small></p><p>Limit results to only those run on the specified framework.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>device_group</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If <code>source</code> is <code>rdc</code>, return results only for the specified device group. Available values are: <ul><li><code>private</code></li><li><code>public</code></li></ul></p></td>
+    </tr>
```
Comment (paweldudzinski) [inline docs/dev/api/insights.md:2888]: Is it worth to indicate that `device_group` only applies to RDC tests?

Comment (LoryApiFortress) [reply docs/dev/api/insights.md:2888]: I did, isn't it enough? "If <code>source</code> is <code>rdc</code>, return results only for the specified device group. "

Comment (paweldudzinski) [reply docs/dev/api/insights.md:2888]: Ah, of course it's enough.

**Review by paweldudzinski — APPROVED** [review]

**Review by aoleszyn — APPROVED** [review]

**Pull request merged**